### PR TITLE
local/staging: use argocd-config 1.1.0

### PR DESCRIPTION
**Pull request by deer-wmde**
> [!IMPORTANT]
> needs https://github.com/wbstack/charts/pull/173

https://phabricator.wikimedia.org/T377585

deploys/uses https://github.com/wbstack/charts/pull/173 in local & staging 

- **add chart versions for argocd**
- **add app-of-apps chart version**

### CODE REVIEW

File: k8s/argocd/local/app-of-apps.values.yaml [hunk 1]
```diff
@@ -1,2 +1,5 @@
 chartVersions:
+  wbaasUi: 0.4.0
```
Comment (tarrow) [inline k8s/argocd/local/app-of-apps.values.yaml:2]: I love being explict here; better than the implicit values from the chart

**Comment by tarrow** [comment]
I think this won't use the new argocd-config chart in any environment? We should need to bump the argocd-config version in helmfile.yaml right?

File: k8s/argocd/local/app-of-apps.values.yaml (hunk 1 above)
Comment (deer-wmde) [reply k8s/argocd/local/app-of-apps.values.yaml:2]: also allows deployment without a new chart

**Comment by deer-wmde** [comment]
> I think this won't use the new argocd-config chart in any environment? We should need to bump the argocd-config version in helmfile.yaml right?

ah right, thats me getting used to not commit the helmfile.yaml ...

File: k8s/helmfile/helmfile.yaml [hunk 2]
```diff
@@ -109,7 +109,7 @@ releases:
   - name: argocd-config
     namespace: argocd
     chart: wbstack/argocd-config
-    version: 1.0.10
+    version: 1.1.0
```
[the remark this answers is not in the document]
Comment (tarrow) [reply k8s/helmfile/helmfile.yaml:112]: yeah, I would say so

[the remark this answers is not in the document]
Comment (deer-wmde) [reply k8s/helmfile/helmfile.yaml]: done

**Review by tarrow — APPROVED** [review]
wheeee lets goooo

**Pull request merged**